subprojects/docs/src/docs/release/notes.md

```diff
@@ -36,13 +41,13 @@ See more information about the [Gradle native project](https://github.com/gradle
 
 ### Recommend C++ projects use new C++ plugins
 
-We introduced new C++ plugins [last year](https://blog.gradle.org/introducing-the-new-cpp-plugins) that we thought were more feature rich and familiar to Gradle users. At the time, we thought they weren't mature enough for serious adoption.
+We introduced [new C++ plugins](https://blog.gradle.org/introducing-the-new-cpp-plugins) last year that we thought were more feature rich and familiar to Gradle users.
```

Comment:
I think this is the confusing part @big-guy We call them the "new" plugins, but they aren't really new in this release.